src/components/chip-widget/operator-input.tsx

```diff
@@ -17,7 +17,7 @@ interface OperatorInputProps {
 const OperatorInput: React.FC<OperatorInputProps> = ({ onApplyOperatorInput, operatorName }) => {
     const inputRef = useRef<HTMLInputElement>(null); // Create a ref for the input element
 
-    const [stringInput, setStringInput] = useState<string>('');
+    const [stringInput, setStringInput] = useState<string | null>(null);
```

Comment:
make null by default so we know if we have a value. Had issue when switching operators and was resetting the value to default.